ament_lint_auto/cmake/ament_lint_auto_find_test_dependencies.cmake

```diff
@@ -17,15 +17,24 @@
 #
 # All found package names are appended to the
 # ``${PROJECT_NAME}_FOUND_TEST_DEPENDS`` variables.
+# :param EXCLUDE: List of linter packages that should be skipped
+# :type EXCLUDE: list of strings
+#
+# :outvar <project_name>_FOUND_TEST_DEPENDS: All found package names
+# :outvar AMENT_LINT_AUTO_EXCLUDE: List of excluded packages
```

Comment:
The doc should mention that all excluded packages are added to this variable and that the variable is being passed to `ament_execute_extensions`.